components/camel-netty/src/test/java/org/apache/camel/component/netty/NettyTCPSyncUDSTest.java

```diff
@@ -19,16 +19,14 @@
 import org.apache.camel.Exchange;
 import org.apache.camel.Processor;
 import org.apache.camel.builder.RouteBuilder;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 /**
- * Add {@code <classifier>linux-x86_64</classifier>} to io.netty:netty-transport-native-epoll dependency to make this
- * test work
+ *
+ * @see <a href="https://netty.io/wiki/native-transports.html">about netty native transport</a>
  */
-@Disabled("Requires native library to load, can be run manually")
```

Comment:
Should it be disabled on Windows?

Reply:
I fill fix it, ok

What kind of note?

Reply:
For the note, I was thinking to provide a mention that `native  transport` is supported on Linux with epoll and Mac with kqueue. Maybe on the `unixDomainSocketPath` https://github.com/apache/camel/blob/0fb44f713f4eedb450970fe4b40a59c2cb2ab6dc/components/camel-netty/src/main/java/org/apache/camel/component/netty/NettyServerBootstrapConfiguration.java#L582

and I understand well that the end-user must add some specific classifier on Mac to make it working (is it right?)

Reply:
Not only on Mac - on Linux too. User must include library with bindings, for example:
```xml
<dependency>
  <groupId>io.netty</groupId>
  <artifactId>netty-transport-native-epoll</artifactId>
  <version>${netty-version}</version>
  <classifier>linux-x86_64</classifier>
</dependency>
```

I will make soon new doc-section about native transports for netty - i think that will be more pretty then any setting description